src/prefect/utilities/importtools.py

```diff
@@ -423,7 +423,7 @@ def safe_load_namespace(source_code: str):
                 logger.debug("Failed to import from %s: %s", node.module, e)
 
     # Handle local definitions
-    for node in ast.walk(parsed_code):
+    for node in parsed_code.body:
```

Comment:
Am I understanding correctly the main change here is `ast.walk` recursively traverses descendant nodes while `parsed_body.code` will only include the top level nodes?

Reply:
Yup, that's correct